Source/Caches/HACache.swift

```diff
@@ -70,6 +70,31 @@ public class HACache<ValueType> {
         )
     }
 
+    /// Create a cache that relies on subscription updates without initial population.
```

Comment:
i think it would be a good idea to break this out into a separate PR, mostly for writing tests distinct for it, since there's a lot in the subscription handling part below.

Reply:
Adding support for "populate-less" then using it in another PR

Reply:
Ok, here they are:
https://github.com/home-assistant/HAKit/pull/48
and 
https://github.com/home-assistant/HAKit/pull/49

I'll be closing this one